spec/unit/harvestdor-indexer_spec.rb

```diff
@@ -153,17 +153,17 @@
     end
   end # whitelist
 
-  it "solr_client initializes the rsolr client using the options from the config" do
+  it 'solr_client initializes the rsolr client using the options from the config' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.